examples/guided.py

```diff
@@ -287,11 +291,16 @@ def perform_installation(mountpoint):
 
 		if installation.minimal_installation():
 			installation.set_hostname(archinstall.arguments['hostname'])
+<<<<<<< HEAD
+			if archinstall.arguments['mirror-region'] != None:
+				installation.set_mirrors(archinstall.arguments['mirror-region']) # Set the mirrors in the installation medium
+=======
```

Comment:
I think this is a automerge issue hehe

Reply:
I've fixed, yeah gits automerge can be a bit funky